feature-libs/product-configurator/rulebased/components/attribute/product-card/configurator-attribute-product-card.component.spec.ts

```diff
@@ -329,6 +329,22 @@ describe('ConfiguratorAttributeProductCardComponent', () => {
 
       expect(button.innerText).toContain('configurator.button.remove');
     });
+
+    it('should show deselection error message when removing required attribute', () => {
+      component.productCardOptions.multiSelect = true;
+      component.productCardOptions.hideRemoveButton = true;
+      setProductBoundValueAttributes(component);
+
+      fixture.detectChanges();
+
+      const button = fixture.debugElement.query(By.css('button.btn'))
+        .nativeElement;
+
+      button.click();
+
+      expect(component.onHandleDeselect).toHaveBeenCalled();
+      expect(component.showDeselectionNotPossible).toBeTruthy;
```

Comment:
```suggestion
       expect(component.showDeselectionNotPossible).toBe(true);
```
toBeTruthy is confusing as it can be applied to all kinds of types, let's use toBe for clarity reasons